scanner/config/config.go

```diff
@@ -1,9 +1,143 @@
 package config
 
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/rs/zerolog"
+	"gopkg.in/yaml.v3"
+)
+
+var (
+	// DefaultConfiguration provides the default values for the scanner configuration.
+	DefaultConfiguration = Config{
+		HTTPListenAddr: ":9443",
```

Comment:
Curious where did this come from? From what I can tell, metrics port tends to be 9090 or so (https://github.com/stackrox/stackrox/blob/4.1.3/pkg/env/metrics.go#L6)

Reply:
A different HTTP server serves Prometheus metrics. The port is configurable through an env called `ROX_METRICS_PORT`.